app/livechat/server/methods/transfer.js

```diff
@@ -32,6 +33,12 @@ Meteor.methods({
 		}
 
 		const guest = LivechatVisitors.findOneById(room.v && room.v._id);
+		transferData.transferredBy = normalizeTransferredByData(Meteor.user() || {}, room);
+		transferData.department = LivechatDepartment.findOneById(transferData.departmentId);
```

Comment:
You don't need to read the model here, you can do it inside the Livechat.transfer.